components/collection/CollectionInfo.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div class="is-flex is-justify-content-space-between flex-direction">
```

Comment:
```suggestion
  <div class="is-flex is-justify-content-space-between">
```

row-direction is a default option

Reply:
I've change the class to mobile-flex-direction-column
i still need it in order to change flex-direction in mobile